.svgrrc

```diff
@@ -1,6 +1,6 @@
 {
 	"replaceAttrValues": {
 	  "#000": "{props.fill}",
-	  "#c00": "{props.fillContrast}",
+	  "#c00": "{props.fillcontrast}",
```

Comment:
Can we please add a comment here for future reference on why we use all lowercase 🙏 ? In case we need to add more props or someone wants to refactor this back to camelCase 😆  Thank you!

Reply:
will do!